tools/keytools/sign.c

```diff
@@ -92,6 +92,8 @@
     #include <wolfssl/wolfcrypt/logging.h>
 #endif
 
+#include "wolfboot/wolfboot.h"
```

Comment:
Ideally we will want to update the sign.py version as well. If not we should document in Signing.md

Reply:
@danielinux please let us know your thoughts on the topic.
If the wolfcrypt-py has the ECC public key decode (I believe it does) I think we should add it.